contracts/sol6/KyberHintHandler.sol

```diff
@@ -165,6 +165,14 @@ abstract contract KyberHintHandler is IKyberHint, Utils5 {
         bytes32[] memory tokenToEthReserveIds,
         uint256[] memory tokenToEthSplits
     ) public view override returns (bytes memory hint) {
+        if (tokenToEthType == TradeType.BestOfAll) {
```

Comment:
Check input splits and Ids are empty and use them for build

Reply:
Build is strict now, throwing error if reserveIds and splits is not empty.